src/main/java/com/roommatematcher/enterprise/dto/Profile.java

```diff
@@ -27,4 +27,12 @@
     private String name;
     @SerializedName("age")
     private int age;
+
+    //this method will add increased persistence functionality
+    //when a profile is added to the database, the profile ID will need to be extracted and placed into persistence storage
+    //without this functionality we will have no foreign keys by which we can tie profile attributes together in the database
```

Comment:
This isn't JavaDoc.  Is it supposed to be?

It looks like a discussion/rationale, which does not belong in source code.  That belongs in the Files Changed section of the pull request.